scripts/amd/run_2gcd.sh

```diff
@@ -0,0 +1,38 @@
+#! /bin/bash
+
+
+## A simple script to run two flash attention forward kernels
+## with batch2-nheads48-d64 on two GPUs in parallel
+
+TRITON_DIR=$(git rev-parse --show-toplevel)
+
+echo $TRITON_DIR
+BENCHMARK_DRIVER=${TRITON_DIR}/scripts/amd/benchmark_flash_attention.py
+
+bs=2
+nheads=48
+d=64
+
+for seqlen in 1024 2048  4096 8192 16384
+do
+    rep=$(echo "163840000 / $seqlen" | bc)
```

Comment:
why `163840000`?

Reply:
I'm trying to make the number of repeats reversely proportional to the seqlen.

Reply:
I've divided it by 16k and reps = 10000, is it worth to run so much repetitions?
I believe we used 250 before,

Reply:
For single GPU benchmarking, I always use 20. But for 2 GCD, since the timer is on the CPU side, we need to run more repeats to amortize the overhead.